.changeset/nice-dingos-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': minor
+---
+
+Deploy to immutablezkevm rari rootstock alephzeroevm chiliz lumia superposition
```

Comment:
```suggestion
Add deployments for chains: immutablezkevm, rari, rootstock, alephzeroevm, chiliz, lumia, and superposition
```

Reply:
nice I'm going to add this as a template to the runbook